packages/firestore/src/core/types.ts

```diff
@@ -30,12 +30,17 @@ export type TargetId = number;
 // they're strings. We should probably (de-)serialize to a common internal type.
 export type ProtoByteString = Uint8Array | string;
 
-/** Describes the online state of the Firestore client */
+/**
+ * Describes the online state of the Firestore client. Note that this does not
+ * indicate whether or not the remote store is trying to connect or not. This is
+ * primarily used for views determining whether or not raise events from cache.
```

Comment:
So it's more nuanced than "whether or not to raise events from cache"...  Maybe:

This is primarily used by the View / EventManager code to change their behavior while offline (e.g. get() calls shouldn't wait for data from the server and snapshot events should set metadata.isFromCache=true).

Reply:
Ok, I've updated the comments. Take a look.